platform.md

```diff
@@ -824,7 +806,7 @@ Usage:
     - The value of `io.buildpacks.lifecycle.metadata` SHALL be modified as follows
       - `run-image.reference` SHALL uniquely identify `<run-image>`
       - `run-image.top-layer` SHALL be set to the uncompressed digest of the top layer in `<run-image>`
-    - The value of `io.buildpacks.stack.*` labels SHALL be modified to that of the new `run-image`
+    - The value of `io.buildpacks.base.*` labels SHALL be modified to that of the new `run-image`
```

Comment:
Should we also update `io.buildpacks.stack.*` labels if they are present? For backwards compat?